packages/server/app/Controllers/Http/UserSettingsController.ts

```diff
@@ -130,4 +134,48 @@ export default class UserSettingsController {
             };
         }
     }
+
+    public async linkGoogleAccount({
+        auth,
+        bouncer,
+        response,
+        request,
+    }: HttpContextContract): Promise<LinkGoogleAccountResponseBody> {
+        const user = auth.user;
+        invariant(
+            user !== undefined,
+            'User must be logged in to update her nickname',
+        );
+        await bouncer.authorize('canLinkGoogleAccount');
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Really neat 👌